src/sentry/relay/projectconfig_cache/redis.py

```diff
@@ -28,11 +30,14 @@ def validate(self):
     def __get_redis_key(self, public_key):
         return f"relayconfig:{public_key}"
 
-    def set_many(self, configs):
+    def __get_redis_rev_key(self, public_key):
+        return f"{self.__get_redis_key(public_key)}.rev"
+
+    def set_many(self, configs: dict[str, Mapping[str, Any]]):
         metrics.incr("relay.projectconfig_cache.write", amount=len(configs), tags={"action": "set"})
 
         # Note: Those are multiple pipelines, one per cluster node
-        p = self.cluster.pipeline()
+        p = self.cluster.pipeline(transaction=False)
```

Comment:
There is no need for this pipeline to be transactional (default value), all keys are independent.